### PR TITLE
zh-trans: /about/community/join/index.md

### PR DESCRIPTION
**Pull request by yuxiaobo96**
Signed-off-by: yuxiaobo <yuxiaobogo@163.com>

ref: https://github.com/servicemesher/istio-official-translation/issues/1344

### CODE REVIEW

File: content/zh/about/community/join/index.md [hunk 1]
```diff
@@ -1,59 +1,53 @@
...
-[working groups](https://github.com/istio/community/blob/master/WORKING-GROUPS.md).
+如果您想参与 Istio 项目，需要了解 [how we do things](https://github.com/istio/community/blob/master/README.md) 并考虑加入我们的
+[工作组](https://github.com/istio/community/blob/master/WORKING-GROUPS.md)。
 {{< /community_item >}}
 
 {{< community_item logo="./servicemesher.svg" alt="ServiceMesher" >}}
-Our Chinese-language documentation is maintained by the
-[ServiceMesher community](http://www.servicemesher.com), join us and get involved!
+中文内容由
+[ServiceMesher 社区](http://www.servicemesher.com) 维护，加入我们并参与进来吧！
```
Comment (rootsongjc) [inline content/zh/about/community/join/index.md]: http://www.servicemesher.com -> https://www.servicemesher.com 
增加了HTTPS

**Comment by yuxiaobo96** [comment]
@rootsongjc 
已修改
THX.

**Comment by rootsongjc** [comment]
image
@yuxiaobo96

**Comment by yuxiaobo96** [comment]
@rootsongjc 
什么意思？是需要翻译成中文还是链接不可用！

**Comment by rootsongjc** [comment]
> @rootsongjc
> 什么意思？是需要翻译成中文还是链接不可用！

需要翻译。

**Comment by yuxiaobo96** [comment]
@rootsongjc 
已修改

**Pull request merged**